stash_engine/lib/stash/download/file.rb

```diff
@@ -7,6 +7,21 @@ module Stash
   module Download
     class File < Base
 
+      attr_accessor :file
+
+      # this downloads a file as a stream from Merritt express
+      def download(file:)
```

Comment:
Should we follow this named parameter standard going forward? (e.g. `download(file: file)` vs. `download(file)`) I'm not opposed to it at all :) Just asking for my own work going forward

Reply:
I think the named parameters are more the normal style since Ruby v2.  We have a bunch of old code (and some old gems and libraries) that use positional parameters instead.  I try to use named parameters when I can.  For one parameter it's probably less of a big deal, but with a long list it becomes easier to know what is being passed in with the named parameters.